vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction.go

```diff
@@ -61,14 +70,26 @@ type PodsEvictionRestrictionFactory interface {
 
 type podsEvictionRestrictionFactoryImpl struct {
 	client                    kube_client.Interface
+	rcInformer                cache.SharedIndexInformer
```

Comment:
Could you please either extend the names here or at least comment them? Looking below it's easy to understand what they mean but without this context it's not clear so let's have some context directly here.

Reply:
Done